src/components/cards/MetadataCard.jsx

```diff
@@ -17,6 +17,12 @@ const badValues = [null, undefined, ''];
 function fieldValueGood(field) {
   const value = field?.value;
   if (badValues.includes(value)) return false;
+  if (Array.isArray(value)) {
+    if (value?.length < 1) return false;
+    const removeNulls = value.filter(entry => entry !== null);
+    if (removeNulls?.length < 1) return false;
```

Comment:
This is for lat/long, although a legit 0,0 lat/long also comes back as [null, null] currently, so it'll be a blindspot for us.